### PR TITLE
The front end of the extension

### PR DESCRIPTION
**Pull request by HarshS1611**
We have build the front end of the chrome extension that takes the image file as an input

### CODE REVIEW

**Comment by HarshS1611** [comment]
I have updated the UI please review it

**Comment by HarshS1611** [comment]
Link of the UI image
Full page view -- https://user-images.githubusercontent.com/81004813/135725223-01e29892-09d0-4636-bda3-5a72fa97580b.png
Only extension  image--
https://user-images.githubusercontent.com/81004813/135725190-64049e2c-89fc-4c60-8165-b10419f62e28.png

**Comment by HarshS1611** [comment]
Icons pixels changed according to the documentation
image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    </div>
+    <div class="row">
+      <div class="column">
+        <div class="form-style">
+          <p style="margin-top: -25px;font-size: medium;">Preview</p>
+        </div>
+      </div>
+      <div class="column1">
+          <div class="image-inside">
+            <div class="upload" style="padding-bottom:20px;padding-top:170px; " >
```
Comment (ohmpatel46) [inline index.html:30]: Try to move all inline styling to a seperate css file.

Comment (HarshS1611) [reply index.html:30]: okk i will do it as soon as possible.

**Review by ohmpatel46 — APPROVED** [review]
Everything seems to be correctly done. 
Suggestion: give a crop/resize option within the extension for the users.
Possible addition: Give an option to set custom backgrounds like simple colors or even other backgrounds.

**Pull request merged**